### PR TITLE
nerves_system_br: bump to v1.21.0

### PR DESCRIPTION
**Pull request by fhunleth**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
-version: 2.1
-
 exec: &exec
   name: build-tools/nerves-system-br
+  version: 1.21.0
+  elixir: 1.13.4-otp-25
   version: 1.20.5
   elixir: 1.13.4-otp-25
```
Comment (jjcarstens) [inline .circleci/config.yml:6]: Looks like you accidentally added the options again rather than overwrite

**Review by jjcarstens — CHANGES_REQUESTED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (fhunleth) [reply .circleci/config.yml]: It's the script. You looked too quickly. :)

File: .circleci/config.yml [hunk 2]
```diff
@@ -1,14 +1,15 @@
...
+  version: 1.21.0
   elixir: 1.13.4-otp-25
 
+version: 2.1
+
 orbs:
   build-tools: nerves-project/build-tools@0.2.2
 
 workflows:
+  version: 2
```
Comment (jjcarstens) [inline .circleci/config.yml:12]:
```suggestion
```
Might need to update your script 😉

File: .circleci/config.yml (hunk 1 above)
Comment (jjcarstens) [reply .circleci/config.yml]: 🏎️ 💨

File: .circleci/config.yml (hunk 2 above)
Comment (fhunleth) [reply .circleci/config.yml:12]: That one is intended.

**Review by jjcarstens — APPROVED** [review]

**Pull request closed without merging**